contracts/staking/test/unit_tests/mixin_scheduler_test.ts

```diff
@@ -0,0 +1,120 @@
+import { blockchainTests, expect } from '@0x/contracts-test-utils';
+import { StakingRevertErrors } from '@0x/order-utils';
+import { BigNumber } from '@0x/utils';
+import { LogWithDecodedArgs } from 'ethereum-types';
+import * as _ from 'lodash';
+
+import { artifacts, TestMixinSchedulerContract, TestMixinSchedulerGoToNextEpochTestInfoEventArgs } from '../../src';
+
+import { constants as stakingConstants } from '../utils/constants';
+
+blockchainTests.resets('MixinScheduler unit tests', env => {
+    let accounts: string[];
+    let owner: string;
+    let authorizedAddress: string;
+    let notAuthorizedAddresses: string[];
+    let testContract: TestMixinSchedulerContract;
+
+    before(async () => {
+        // Create accounts
+        accounts = await env.getAccountAddressesAsync();
+        [owner, authorizedAddress, ...notAuthorizedAddresses] = accounts;
```

Comment:
It turns out that none of these accounts are actually being used. Let's remove them and remove all of the `let` variables above.